README.md

```diff
@@ -242,6 +242,33 @@ const client = new faunadb.Client({
 })
 ```
 
+## Known issues
+
+### Using with Cloudflare Workers
+
+Cloudflare Workers has neither XMLHttpRequest nor fetch in global scope.
+Therefore, cross-fetch package is unable to inject fetch, and throws an error.
+fetch function is injected via closure, so the workaround would be to pass
+the fetch objects when initiating the FaunaDB client config. Cloudflare also
```

Comment:
Objects? Do you mean "function"? If not, it would be good to provide a bit more context here.

```suggestion
the fetch objects when initiating the FaunaDB client config. Cloudflare Workers also
```

Reply:
yes, it's about function

Reply:
That means that `objects` needs to be replaced with `function`.